src/utils/update/company.profile.mapper.to.oversea.entity.ts

```diff
@@ -0,0 +1,30 @@
+import { CompanyProfile } from "@companieshouse/api-sdk-node/dist/services/company-profile/types";
+import { Entity } from "model/entity.model";
+
+export const mapCompanyProfileToOverseasEntityToDTOUpdate = (cp: CompanyProfile): Entity => {
+  if (!cp){
```

Comment:
Better to fail than to return and empty Entity. Remove if

Reply:
But shouldn't we be handling the fail? An undefined data will generate a 500 error and need to demonstrate unit test in place to handle it. Checking through existing CH mapping, similar approach is in place

Reply:
Why are you expecting it to fail (be undefined or null)? We should always get back a CompanyProfile as the SDK should always return a result.